Snakefiles/workflow/scripts/compute_powerlaw_fit_from_hic.py

```diff
@@ -22,7 +22,7 @@ class formatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawTextHelpForm
     readable = argparse.FileType('r')
     parser.add_argument('--hicDir', help="Directory containing observed HiC KR normalized matrices. File naming and structure should be: hicDir/chr*/chr*.KRobserved")
     parser.add_argument('--outDir', help="Output directory")
-    parser.add_argument('--hic_type', default = 'juicebox', choices=['juicebox','bedpe', 'avg'], help="format of hic files")
+    parser.add_argument('--hic_type', default = 'juicebox', choices=['juicebox','bedpe'], help="format of hic files")
```

Comment:
the 'avg' parameter (and later code handling this parameter) is probably important for the ENCODE predictions; the 'avg' probably refers to the specially formatted cell-type average Hi-C that kristy made for this ENCODE set of predictions

Reply:
I guess you probably haven't worked on this particular file (compute_powerlaw_fit_from_hic.py).  It looks like all of the Kristy's encode v2 branch changes should be copied into dev branch

Reply:
Yeah I haven't touched this file on this `dev` branch